.github/workflows/test-tgc.yml

```diff
@@ -25,6 +25,8 @@ on:
       sha:
         description: "The commit SHA in magic-modules repository where the status result will be posted"
         required: true
+      initiator_id:
+        description: "Metadata about the caller who initiated the dispatch event"
```

Comment:
Maybe reword to "Identity of the workflow dispatch caller" and possibly give example values.

Reply:
Changed to `caller_id`. Avoiding build id since that suggests a parent, which I want to avoid.